piet/src/text.rs

```diff
@@ -29,6 +28,59 @@ pub trait Text: Clone {
     /// [`FontFamily`]: struct.FontFamily.html
     fn font_family(&mut self, family_name: &str) -> Option<FontFamily>;
 
+    /// Load the provided font data and make it available for use.
+    ///
+    /// This method takes font data (such as the contents of a file on disk) and
+    /// attempts to load it, making it subsequently available for use.
+    ///
+    /// If loading is successful, this method will return a [`FontFamily`] handle
+    /// that can be used to select this font when constructing a [`TextLayout`].
+    ///
+    /// # Notes
+    ///
+    /// ## font familes and styles:
+    ///
+    /// If you wish to use multiple fonts in a given family, you will need to
```

Comment:
So this makes me a bit uncomfortable. The *right* thing to do is load multiple styles in a single family, but I understand that's a lot harder. A note that we plan to change the API would be sufficient for this cycle.

Reply:
maybe my wording is bad. A single file that contains multiple styles *will* work; you just can't address the individually loaded styles using the handle you get back, you need to resolve the style by setting italic/weight as appropriate.

One thing that we don't make easy, here, is accessing a condensed style, but I'm okay with that for now.